back/src/core/security/crypto.ts

```diff
@@ -1,14 +1,11 @@
 import * as crypto from "node:crypto"
 require("dotenv").config()
 
-const keyLength = Number(process.env.CRYPTO_KEY_LENGTH!)
-const defaultSalt = process.env.CRYPTO_SECRET!
-
 export const generateRandomSalt = (): string => {
-  return crypto.randomBytes(keyLength).toString("hex")
-}
-
-
+    return crypto.randomBytes(keyLength).toString("hex")
+  },
+  keyLength = Number(process.env.CRYPTO_KEY_LENGTH!),
+  defaultSalt = process.env.CRYPTO_SECRET!
```

Comment:
da uma olhada nesses foruns que comentam sobre as vantagens e desvangans de declara variaveis em uma unica linha:

https://www.reddit.com/r/learnprogramming/comments/9zgvlk/is_declaring_multiple_variables_in_the_same_line/
https://stackoverflow.com/questions/694102/declaring-multiple-variables-in-javascript

<h4><a href="https://www.notion.so/taqtile/Engineers-ae071e6916d3430ab2637982b9436fcb">Links uteis no notion da Taq</a></h4>

Reply:
Minha recomendação: considerando que a forma mais legivel e "maintanable" possivel seja a declaracao de variaveis em linhas, eu sugiro manter o jeito que estavamos fazendo